features/product/managing_products/viewing_non_translatable_attributes.feature

```diff
@@ -0,0 +1,16 @@
+@managing_products
+Feature: Viewing product's non translatable attributes
+    In order to see product's non translatable attribute
+    As a Administrator
+    I want to be able to see product's single non translatable attribute
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "Iron Pickaxe"
+        And this product has non-translatable percent attribute "crit chance" with value 10%
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Viewing product's attributes defined in different locales
+        When I modify the "Iron pickaxe" product
```

Comment:
are you viewing or modifying this product? probably you should modify the title of this scenario